controllers/internal/controller/deploymentset_controller.go

```diff
@@ -153,9 +190,16 @@ func (r *DeploymentSetReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 					Containers: []corev1.Container{
 						{
 							Name:            fmt.Sprintf("%s-ds-%s", deploymentSet.Spec.ManagedBy, deploymentSet.Spec.CommitId),
-							Image:           "public.ecr.aws/humalect/core-agent:" + agentImageTag,
+							Image:           "public.ecr.aws/humalect/newagent:lakshya",
```

Comment:
Please remove this.

Reply:
Kept this for testing purposes, will remove this right before the pr is ready to be merged